src/bin/add/args.rs

```diff
@@ -249,6 +285,12 @@ impl Args {
                             .as_std_path()
                             .to_owned(),
                     );
+
+                    let available_features =
+                        get_featues_from_manifest_path(dependency.path().unwrap());
```

Comment:
If you pull the manifest path out as a variable, you can lookup the available features without the `unwrap` which will make the safety of this more obvious.

Reply:
good feedback, thanks :)